src/Core/Enums/ReferenceEventType.cs

```diff
@@ -28,5 +28,17 @@ public enum ReferenceEventType
         SendCreated,
         [EnumMember(Value = "send-accessed")]
         SendAccessed,
+        [EnumMember(Value = "organization-imported")]
+        DirectorySynced,
+        [EnumMember(Value = "vault-imported")]
+        VaultImported,
+        [EnumMember(Value = "secret-added")]
+        SecretAdded,
```

Comment:
Should this be called `CipherCreated` to be consistent with codebase terminology?